mix.exs

```diff
@@ -50,11 +50,11 @@ defmodule Rajska.MixProject do
   defp deps do
     [
       {:ex_doc, "~> 0.19", only: :dev, runtime: false},
-      {:credo, "~> 1.5.0", only: [:dev, :test], runtime: false},
-      {:absinthe, "~> 1.4.0 or ~> 1.5.4 or ~> 1.6.0"},
+      {:credo, "~> 1.6.0", only: [:dev, :test], runtime: false},
+      {:absinthe, "~> 1.5.4 or ~> 1.6.0 or ~> 1.7"},
```

Comment:
similar question to the comment above about the elixir version, do we need to drop support for absinthe 1.4? If it still works, better to keep it